examples/visual-tests/amp-story-player/back-button.html

```diff
@@ -0,0 +1,55 @@
+<html>
+  <head>
+    <title>Story Player (Non-AMP)</title>
+    <script>
+      // TODO(): Remove this when unit tests for player are fixed.
+      __AMP_VISUAL_TEST = true;
+    </script>
+    <script async src="../../../dist/amp-story-player.js"></script>
+    <link href="../../../build/css/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
```

Comment:
Not sure if this is related, but try to include these files like you would in a normal AMP page (`https://cdn.ampproject.org/amp-story-player-v0.js` and `.css`) - the internal server should rewrite them correctly both locally and on Travis. Try this and let me know if this makes things better :)

Reply:
Tried, did not work. [Build for change urls commit](https://percy.io/ampproject/amphtml/builds/6149656)

Reply:
Error was script should have been the minified version.